packages/utils/test/unit/fetch.test.ts

```diff
@@ -18,7 +18,6 @@ describe("FetchError", () => {
   }[] = [
     {
       id: "Bad domain",
-      // Use random bytes to ensure no collisions
```

Comment:
please try to remove unnecessary modifications

Reply:
Done. 
@nflaig, you can check it out now.